int32 aruco_id
---
geometry_msgs/Pose pose